Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser.py

```diff
@@ -159,8 +165,19 @@ def test_function(client, _):
        Performs basic GET request to check if the API is reachable and authentication is successful.
        Returns ok if successful.
        """
+    response = 'ok'
+    if demisto.params().get('self_deployed', False):
+        response = '```✅ Success!```'
+        if demisto.command() == 'test-module':
```

Comment:
why is this condition needed?

Reply:
we test that the test-module commands returns ok in the test_integration, it fails on the emoji.

Reply:
not sure i follow.. what is "the test_integration" ?

Reply:
https://app.circleci.com/pipelines/github/demisto/content/26640/workflows/cabc2c3f-9d0a-4d12-94bb-f493a2460c8e/jobs/114100

Reply:
how come it even gets here? it should not pass the condition `if demisto.params().get('self_deployed', False)`

Reply:
It was in an old config, i will change and run it again.
commit: https://github.com/demisto/content/commit/21e3d10171b29b85ca0ff0e11c6724e20d3145f3
circle build: https://app.circleci.com/pipelines/github/demisto/content/27141/workflows/11d0d343-1c05-4a16-bc29-861294a932e5

Reply:
```
Start of Instance Testing ("Test" button) (Post-update)
Testing ("Test" button) for instance "ms_graph_user_dev_test_48b633ab-e32b-45fa-bc6a-c57cd522dd5b" of integration "Microsoft Graph User".
trying to connect.
Test integration failed - server: https://ec2-44-234-253-122.us-west-2.compute.amazonaws.com.
Failure message: ```✅ Success!``` (85)
Testing ("Test" button) for instance "ms_graph_user_prod_test_0e6ecdd3-fe25-42d3-8397-7c68ad07d965" of integration "Microsoft Graph User".
```

I'll revert.